editor/src/components/inspector/sections/component-section/data-selector-columns.tsx

```diff
@@ -232,6 +225,16 @@ const DataSelectorFlexColumn = styled(FlexColumn)({
   borderRight: `1px solid ${colorTheme.subduedBorder.cssValue}`,
 })
 
+export const DataPickerRow = styled(FlexRow)({
+  alignSelf: 'stretch',
+  justifyContent: 'space-between',
+  fontSize: 10,
+  borderRadius: 4,
+  height: 24,
+  padding: 5,
+  cursor: 'pointer',
```

Comment:
we should only show a pointer when the row is actually clickable, this was a bit confusing when I was trying it

Reply:
ah good point!